lib/windows/handle.rb

```diff
@@ -9,7 +9,7 @@ module Handle
 
     private
 
-    INVALID_HANDLE_VALUE           = 0xFFFFFFFF
+    INVALID_HANDLE_VALUE           = (1<<['a'].pack('p').length*8)-1
```

Comment:
> regardless of whether i'm using 32-bit or 64-bit Ruby

Really? I got 0xFFFFFFFFFFFFFFFF when I use Ruby 2.3 x64
AppVeyor CI passes for i386 and x64 Rubies even if using this code: https://ci.appveyor.com/project/cosmo0920/windows-pr/build/4